### ISSUE TITLE
Implement rich text

### ISSUE THREAD

**Issue by mariobehling** [fossasia/badgemagic-app#272]
The screen could have options like font, font size etc. Please implement it.

![Screenshot_2019-05-07-01-20-24-830_cn huidu huiduapp](https://user-images.githubusercontent.com/1583873/57242818-18b57b80-7067-11e9-9e75-211010d8a324.png)

**Comment by adityastic** [comment]
@mariobehling @iamareebjamal

**Comment by akri16** [comment]
I can implement some of these features but, doesn't color depend on the hardware?
Please correct me if I`m wrong

**Comment by adityastic** [comment]
> I can implement some of these features but, doesn't color depend on the hardware?
> Please correct me if I`m wrong

Yes color does depend on the hardware. I don't have a doc of the hardware and the apis it produces. We can only test it with the original application (lsled) which you can find on the play store and using wireshark. What 'some' features are you planning to implement? Because if you see my last comment, there will be a lot of difficulty viewing style change due to the fixed amount of pixels we have.

**Comment by akri16** [comment]
Great! thank you for the clarification. I'll work on this after I resolve the current issue I'm assigned to. I'll have to research a bit on it before I could give you an extensive list of Whats and Hows. I'll run everything by you before I start working on it :)

**Comment by mariobehling** [comment]
> This is one of the limitations, each letter currently occupies 8 bits with 1 bit always 0. We don't have enough space to implement font styles in the badge

It is correct that the display of different fonts is limited and that font features will be scaled down massively on the display of the badge. However, the proprietary apps, e.g. on Windows have font options and the display does change. So, an implementation is possible.
